spring-cloud-stream-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -152,18 +152,51 @@ Setting up the two properties correctly on scale up scenarios is important for a
 general (see below), and they are always required by certain types of binders (e.g. the Kafka binder) in order to
 ensure that data is split correctly across multiple consumer instances.
 
-==== Advanced Binding Properties
+==== Binding Properties
 
 The input and output destination names are the primary properties to set in order to have Spring Cloud Stream
 applications communicate with each other as their channels are bound to an external message broker automatically.
 However, there are a number of scenarios where it is required to configure other attributes besides the destination
 name. This is done using the following naming scheme:
 `spring.cloud.stream.bindings.<channelName>.<attributeName>=<attributeValue>`. The `destination` attribute is one such
-example: `spring.cloud.stream.bindings.input.destination=foo`. A shorthand equivalent can be used as follows:
-`spring.cloud.stream.bindings.input=foo`, but that shorthand can only be used only when there are no other attributes
-to set on the binding. In other words,
-`spring.cloud.stream.bindings.input.destination=foo`,`spring.cloud.stream.bindings.input.partitioned=true` is a valid
-setup, whereas `spring.cloud.stream.bindings.input=foo`,`spring.cloud.stream.bindings.input.partitioned=true` is not.
+example: `spring.cloud.stream.bindings.input.destination=foo`.
+
+The `group` property (spring.cloud.stream.bindings.<channelName>.group) is used to provide a unique name the binding
```

Comment:
wouldn't refer to this as "unique"... rather just: "is used to provide the name of a consumer group."